src/helpers/figures/charts/chart_common.ts

```diff
@@ -532,10 +532,19 @@ export function interpolateData(
     return [];
   }
   const labelMin = Math.min(...labels);
```

Comment:
We don't need this anymore in the case of a logarithmic trend line, so it should be better to have it in a separated if (see below :) )